components/bms_worker/src/Environment.c

```diff
@@ -162,8 +162,8 @@ static void Environment10Hz_PRD()
         {
             hs_state = DONE;
 
-            ENV.values.board.ambient_temp = hs_chip.data.temp;
-            ENV.values.board.rh           = hs_chip.data.rh;
```

Comment:
for hs_chip.data.temp and hs_chip.data.rh , rh and temp are of type int in their respective structs and files. I know it is redundant to divide by 100 or 10 an int because the precision won't improve. I wasn't sure if I should change it in their file HW_HS4011.h/c